pkg/vault/contracts/authorizer/TimelockAuthorizer.sol

```diff
@@ -619,7 +638,26 @@ contract TimelockAuthorizer is IAuthorizer, ReentrancyGuard {
     }
 
     /**
-     * @notice Schedules an execution to set the delay for granting permission over `actionId` to `newDelay`.
+     * @notice Schedules an execution to set the delay for granting permission over `actionId` to `newDelay`. This makes
+     * it impossible to grant permission to execute `actionId` without an immutable public on-chain committment for the
+     * granting at least `newDelay` seconds in advance.
+     *
+     * Critical actions that are expected to be performed by smart contracts are typically subject to said grant delays
+     * to allow for public scrutiny of new contracts that are granted the permission.
+     *
+     * How long it will take to make this change will depend on the current and new grant delays: if increasing by more
+     * than 5 days, then the time difference between the grant delays must pass. Otherwise, the minimum delay change
+     * execution delay of 5 days must pass instead.
+     *
+     * Only `executors` will be able to execute the scheduled action, unless `executors` is an empty array, in which
+     * case any account can execute it.
+     *
+     * Avoid scheduling multiple grant delay changes for the same action at the same time, as this makes it harder to
+     * reason about the state of the system. If there is already a scheduled grant delay change and there is a desire to
+     * change the future grant delay to some other value, cancel the first scheduled change and schedule a new one.
+     *
+     * Only root can call this function, but other accounts may be granted permission to cancel the scheduled execution
+     * (including global cancelers).
```

Comment:
```suggestion
     * @notice Schedules an execution to set the delay for granting permission over `actionId` to `newDelay`. This makes
     * it impossible to grant permission to execute `actionId` without an immutable public on-chain commitment for the
     * granting at least `newDelay` seconds in advance.
     *
     * Critical actions that are expected to be performed by smart contracts are typically subject to such grant delays
     * to allow for public scrutiny of new contracts that are granted the permission.
     *
     * How long it will take to make this change will depend on the current and new grant delays: if increasing by more
     * than 5 days, then the time difference between the grant delays must pass. Otherwise, the minimum delay change
     * execution delay of 5 days must pass instead.
     *
     * Only `executors` will be able to execute the scheduled action, unless `executors` is an empty array, in which
     * case any account can execute it.
     *
     * Avoid scheduling multiple grant delay changes for the same action at the same time, as this makes it harder to
     * reason about the state of the system. If there is already a scheduled grant delay change and there is a desire to
     * change the future grant delay to some other value, cancel the first scheduled change and schedule a new one.
     *
     * Only root can call this function, but other accounts may be granted permission to cancel the scheduled execution
     * (including global cancelers).
```